contracts/CurrencyNetwork.sol

```diff
@@ -478,27 +563,38 @@ contract CurrencyNetwork is CurrencyNetworkInterface, Ownable, Authorizable, Des
             result.feesOutstandingB = account.feesOutstandingA;
             result.feesOutstandingA = account.feesOutstandingB;
             result.mtime = account.mtime;
-            result.balance = -account.balance;
+            result.balance = -account.balance; // balance is value receiver owes sender
         }
         return result;
     }
 
+    // Provides the abstraction of whether a < b or b < a.
     function _storeAccount(address _a, address _b, Account account) internal {
         Account storage acc = accounts[uniqueIdentifier(_a, _b)];
         if (_a < _b) {
             acc.creditlineGiven = account.creditlineGiven;
             acc.creditlineReceived = account.creditlineReceived;
-            acc.interestRateGiven = account.interestRateGiven;
-            acc.interestRateReceived = account.interestRateReceived;
+            if (! customInterests) {
```

Comment:
Need to discuss this.
Basically this is a safety net enforcing correct behaviour if some other person come to modify the code or use the functions later. I have strict requires to every public / external functions, so this if is currently useless, but makes sure the data stored is trustworthy, in case some other functions come to use _storeAccount() later.
I believe that is not what should be done but thought that is what you wanted, referring to the discussion as to where we should validate inputs.

Reply:
If this is a safty net, I think it is best here to just assert.